plugins/task-resources/src/components/kanban/KanbanPanelEmpty.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+    import { IconAdd, Button, EditBox, ActionIcon, IconClose } from '@anticrm/ui'
+    import { createEventDispatcher } from 'svelte';
+    import tracker from '../../plugin';
+
+    let isAdding = false
+    const dispatch = createEventDispatcher()
+    const newList = {
+        title: '',
+        color: ''
+    }
+
+    async function onAdd () {
+        if (!newList.title) return;
+        dispatch('add', newList)
+        isAdding = false
+    }
+</script>
+
+<div  class="panel-container step-lr75">
+    {#if isAdding}
+        <EditBox
+            bind:value={newList.title}
+            maxWidth={'19rem'}
+            placeholder={tracker.string.NewListPlaceholder}
+        />
+        <div class="list-add-controls">
+            <Button
+                icon={IconAdd}
+                label={tracker.string.AddList}
+                justify={'left'}
+                on:click={() => { onAdd() }}
+            />
+            <ActionIcon
+                icon={IconClose}
+                size={'large'}
+                action={() => { isAdding = false }}
+            />
+        </div>
+    {:else}
+        <Button
+        icon={IconAdd}
+        label={tracker.string.NewList}
+        justify={'left'}
+        width={'100%'}
+        on:click={() => { isAdding = true }}
+        />
+    {/if}
+</div>
+
+<style lang="scss">
+    .panel-container {
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+        width: 20rem;
+        height: fit-content;
+        background-color: transparent;
+        border: 1px solid transparent;
+        border-radius: .25rem;
+        padding: 4px;
```

Comment:
Please do not mix px and rem sizes. Also please use flex if possible without explicit with. global.scss could be source for some existing styles.

Reply:
Fixed